modules/claims_api/app/swagger/claims_api/v1/swagger.json

```diff
@@ -8944,14 +8962,32 @@
                                 "description": "Service Branch for the veteran.",
                                 "type": "string",
                                 "enum": [
-                                  "AIR FORCE",
-                                  "ARMY",
-                                  "COAST GUARD",
-                                  "MARINE CORPS",
-                                  "NAVY",
-                                  "OTHER"
+                                  "Air Force",
+                                  "Air Force Academy",
+                                  "Air Force Reserves",
+                                  "Air National Guard",
+                                  "Army",
+                                  "Army Air Corps",
+                                  "Army Air Corps or Army Air Force",
+                                  "Army National Guard",
+                                  "Army Reserves",
+                                  "Coast Guard",
+                                  "Coast Guard Academy",
+                                  "Coast Guard Reserves",
+                                  "Marine Corps",
+                                  "Marine Corps Reserves",
+                                  "Merchant Marine",
+                                  "National Oceanic and Atmospheric Administration",
+                                  "Naval Academy",
+                                  "Navy",
+                                  "Navy Reserves",
+                                  "Other",
+                                  "Public Health Service",
+                                  "Space Force",
+                                  "US Military Academy",
+                                  "Women's Army Corps"
```

Comment:
I'm talking with Kayla to try and nail down what the best approach will be but, if we're gonna provide all of these new values, we're going to have to update the PDF generation logic so that it knows which checkbox to select when the PDF is being generated.